.travis.yml

```diff
@@ -1,3 +1,11 @@
 language: minimal
 install:
+  - tar zxvf data/FASTQ.tgz
```

Comment:
Would it make sense to lump these bits into a `makefile`, or easier to defer that to a future refactor?